lib/msal-node/src/index.ts

```diff
@@ -1,2 +1,5 @@
 export { PublicClientApplication } from './client/PublicClientApplication';
 export { ConfidentialClientApplication } from './client/ConfidentialClientApplication';
+
+// crypto
+export { CryptoOps } from './crypto/CryptoOps';
```

Comment:
do we need to export these?

Reply:
This is placeholder as I cannot compile it otherwise, removed in the following PR since `PublicClientApplication` is fleshed out there.